<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminRFID</class>
 <widget class="QDialog" name="AdminRFID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminRFID</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AdminRFID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPortList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>RFID Reader Communications Port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>No card has been detected yet</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>ID is copied to clipboard as soon as it appears.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbScan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rescan Ports</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbConnect</sender>
   <signal>clicked()</signal>
   <receiver>AdminRFID</receiver>
   <slot>onConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbScan</sender>
   <signal>clicked()</signal>
   <receiver>AdminRFID</receiver>
   <slot>scanPorts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnect()</slot>
  <slot>scanPorts()</slot>
 </slots>
</ui>
